patches/gcc/linaro-6.3-2017.02/951-bionic-ndk.patch

```diff
@@ -0,0 +1,58 @@
+commit d38d37bdfe24b7ce1bdcb55642fb6b904718e68f
+Author: Howard Chu <hyc@symas.com>
+Date:   Tue Apr 25 19:02:18 2017 -0700
+
+    Fix ctype for newer NDK headers
+
+diff --git a/libstdc++-v3/config/os/bionic/ctype_base.h b/libstdc++-v3/config/os/bionic/ctype_base.h
+index 33978f3..c36e63c 100644
+--- a/libstdc++-v3/config/os/bionic/ctype_base.h
++++ b/libstdc++-v3/config/os/bionic/ctype_base.h
+@@ -28,6 +28,18 @@
+ 
+ // Information as gleaned from /usr/include/ctype.h
+ 
++// _CTYPE prefix was added in NDK r14 unified headers
```

Comment:
What are you talking about? This particular file is in "os/bionic" - it is *only* used for Android builds.